### DISCUSSION TITLE
Ignore this

### DISCUSSION THREAD

**Discussion by ghosty141** [joaotavora/eglot#1286] [Q&A]
The error is not on eglots side but in clangd. Ignore this!

**Comment by ghost** [comment]
You can custom variable 'project-vc-extra-root-markers'
```emacs-lisp
(add-to-list "compile_commands.json" project-vc-extra-root-markers)
```

**Reply by ghosty141** [reply]
This does not help since while the symlinks live under the root of the project, the actual path of the libraries is above the root folder, thats why I have to reside to this workaround. I believe thats the reason why eglot has problems with that setup but I'm not 100% sure.
